### DISCUSSION TITLE
unraid docker安装paopaodns后data目录下没有任何文件。

### DISCUSSION THREAD

**Discussion by messiah27** [kkkgo/PaoPaoDNS#55] [问答和帮助]
纯新手小白，基本完全不懂。docker正常安装paopaodns，/data目录也已经映射到存储位置，按照博客中的nslookup -type=TXT whoami.ds.akahelp.net ip 命令查看也是宽带ip看起来都是正常的，但是data目录下没有任何文件，全局搜索也没有相关的conf文件或是数据库文件，dnscrypt-resolvers文件夹也不存在，各种yaml和txt文件也找不到，不知道是什么问题。。顺便大神可以做个视频什么的讲解一下食用方法么，很眼馋这个东西。目前是routeros用static列表进行的dns分流，导入一次耗时巨大。上游dns分别用两个adg进行国内外分开并分别填入的dot/doh dns列表。爬墙用的是pve CT容器里的clashmeta，通过bgp隧道宣告路由表进行的流量分流。但是在使用自己的vps节点时查看ipleak会出现dns泄漏，所以很想试试能不能用这个东西解决一下。。

**Comment by kkkgo** [comment]
参考 https://github.com/kkkgo/PaoPaoDNS/discussions/52

**Comment by kkkgo** [comment]
参考 https://github.com/kkkgo/PaoPaoGateWay/discussions/23

**Reply by kkkgo** [reply]
参考 https://github.com/kkkgo/PaoPaoGateWay/discussions/7
